gz-common6.yaml

```diff
@@ -12,6 +12,10 @@ repositories:
     type: git
     url: https://github.com/gazebosim/gz-math
     version: gz-math8
+  gz-msgs:
+    type: git
+    url: https://github.com/gazebosim/gz-msgs
+    version: gz-msgs11
```

Comment:
is this needed? does common depend on msgs?

Reply:
Not needed. Removed it in [866619d](https://github.com/gazebo-tooling/gazebodistro/pull/173/commits/866619db4e7a2ee16dd83131f97503fbc97ef8d2)